Exception/MissingDependencyException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony2 PhoneNumberBundle.
+ *
+ * (c) University of Cambridge
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Misd\PhoneNumberBundle\Exception;
+
+use RuntimeException;
+
+/**
+ * Exception thrown if a dependency is required.
+ */
+class MissingDependencyException extends RuntimeException
```

Comment:
Not sure this worth a dedicated class. Does something could catch surch exception to perform a specific business logic?